scripts/newreference.py

```diff
@@ -22,6 +23,12 @@ def new_reference(referencefile, outgenbank, outfasta, gene):
                                  qualifiers=ref_source_feature.qualifiers)
     record.features.append(source_feature)
 
+    # If user provides a --gene 'some name' is not found, print a warning and use the entire genome.
+    # Otherwise do not print a warning.
+    if(gene != None and startofgene==None and endofgene==None):
```

Comment:
_non-blocking_ 

FYI, we usually use `is` when comparing with `None` values in Python. 
[PEP 8](https://peps.python.org/pep-0008/#programming-recommendations) recommends
> Comparisons to singletons like None should always be done with is or is not, never the equality operators.

```suggestion
    if(gene is not None and startofgene is None and endofgene is None):
```